### ISSUE TITLE
Create Automated Patching using Systems Manager.

### ISSUE THREAD

**Issue by MichaelFriedenthal-NOAA** [NOAA-OWP/hydrovis#615]
Setup System Manager to automate patching across they HydroVIS accounts.  It should only apply patches to instances tagged. Setup the ability to patch in Dev/Test on Sunday, Prod on Tuesday in a single region and then have the same schedule the following weeks in the other region to avoid downtime.

**Comment by MichaelFriedenthal-NOAA** [comment]
This has been deployed, but we need to redeploy this for two reasons:
1.  We need to submit a ticket prior to deployment per OCIO
2.  We need to make an update to the schedule.  The patching schedule initially discussed and implemented was  

1st Sunday - Dev/Test
1st Tuesday - Prod
2nd Sunday - Dev/Test
2nd Tuesday  - Prod.

We realized this month, the first Sunday comes after the first Tuesday (Sunday was 12/31 so Tuesday is 1/2 and Sunday is 1/6), which wouldn't let us test on Dev/Test before patching Prod.  To address this issue, I updated the documentation and will work with Diwakar (since he has access to the Master Payer Account) to change the schedule to as follows:

1st Sunday - Dev/Test
2nd Tuesday - Prod
2nd Sunday - Dev/Test
3rd Tuesday  - Prod.
